jscomp/core/lam_compile.mli

```diff
@@ -25,6 +25,6 @@
 (** Compile single lambda IR to JS IR  *)
 
 val compile_recursive_lets :
-  Lam_compile_context.t -> (Ident.t * Lam.t) list -> Js_output.t
+  string -> Js_packages_info.module_system -> Lam_compile_context.t -> (Ident.t * Lam.t) list -> Js_output.t
```

Comment:
With so many arguments, how about using named arguments so the code is more readable.